src/main/java/jenkins/plugins/git/GitSCMFileSystem.java

```diff
@@ -282,11 +282,16 @@ public SCMFileSystem build(@NonNull Item owner, @NonNull SCM scm, @CheckForNull
             if (rev != null && !(rev instanceof AbstractGitSCMSource.SCMRevisionImpl)) {
                 return null;
             }
-            TaskListener listener = new LogTaskListener(LOGGER, Level.FINE);
             GitSCM gitSCM = (GitSCM) scm;
             UserRemoteConfig config = gitSCM.getUserRemoteConfigs().get(0);
             BranchSpec branchSpec = gitSCM.getBranches().get(0);
             String remote = config.getUrl();
+            LogTaskListener listener = new LogTaskListener(LOGGER, Level.FINE);
```

Comment:
Why the change from TaskListener to LogTaskListener as the type?

Reply:
Didn't notice the type changed I'll revert that bit